app/dao/users_dao.py

```diff
@@ -82,6 +82,14 @@ def delete_user_verify_codes(user):
     db.session.commit()
 
 
+def count_user_verify_codes(user):
+    query = db.session.query(
+        func.count().label('count')
+    ).filter(VerifyCode.user == user,
+             VerifyCode.expiry_datetime <= datetime.utcnow()).one()
```

Comment:
don't we specifically want to only capture verify codes where they _haven't_ expired and also where code_used = False?

otherwise if i log in and out (legitimately) a lot of times earlier in the day, it'll refuse to send